pre_run.py

```diff
@@ -12,6 +12,7 @@
     "payout_ratio_max_val": 0.7,
     "debt_return_time_max_val_by_ebitda": 5.0,
     "debt_return_time_max_val_by_income": 5.0,
+    "years_dividends_growth": 5,
```

Comment:
what does that parameter mean?
that the dividend growth will calculate 5 years back?

Reply:
Exactly